superdesk/celery_app.py

```diff
@@ -168,18 +168,20 @@ def update_key(key, flag=False, db=None):
 
 def _update_subtask_progress(task_id, current=None, total=None, done=None):
     redis_db = redis.from_url(celery.conf['CELERY_RESULT_BACKEND'])
```

Comment:
it can't be in try/finally because then it would try to close it even if it fails on that line